### ISSUE TITLE
Suppress printing/file i/o option

### ISSUE THREAD

**Issue by ncasetti7** [eljost/pysisyphus#285]
Hi devs,

I was wondering if it would be possible to add an option to optimizers/interpolators (or maybe there is an option and I'm just not seeing it) to avoid printing/writing to files. I've been using pysisyphus with a neural network potential, and I'm trying to get some speedup by parallelizing with multiprocessing, but I'm running into issues with race conditions. It would be very convenient to turn off all file i/o and printing.

Thanks in advance!

**Comment by eljost** [comment]
Dear Nicholas,

thanks for your interest in pysisyphus. Currently, this is not possible. I made some changes to the `Optimizer` base-class on the `dev`-branch to control printing better, but this is not yet merged into `master`.

What do you mean with race-conditions and supressing file I/O? Regarding the latter there is also some progress on the `dev`-branch where for example only the files of the previous/latest calculation can be kept.

Do you use multiprocessing in one pysisyphus-process or do you run multiple pysisyphus-processes in parallel?

It seems possible to suppress STDOUT in multiprocessing, as described [https://stackoverflow.com/a/66266293](here).
